library/ix-dev/community/chia/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+name: chia
+description: Chia is a modern cryptocurrency built from scratch, designed to be efficient, decentralized, and secure.
+annotations:
+  title: Chia
```

Comment:
If `title` is not specified at top level dict,we capitalize the first letter of `name` but ideally we should be having it there jfyi

Reply:
Only reason moved it under annotation is because of this:
While technically Helm does not "hard" exit if there are other fields there, I'm not sure how it will be handled in future versions.

https://helm.sh/docs/topics/charts/#the-chartyaml-file
```
As of [v3.3.2](https://github.com/helm/helm/releases/tag/v3.3.2), additional fields are not allowed. The recommended approach is to add custom metadata in annotations.
```